litmus-portal/frontend/src/components/InputField/styles.ts

```diff
@@ -12,14 +12,18 @@ const useStyles = makeStyles((theme) => ({
   },
 
   inputArea: {
-    width: '20rem',
-    marginTop: theme.spacing(3.5),
+    width: theme.spacing(45),
```

Comment:
You can use rem for width.

Reply:
I think theme.spacing is a better choice for width and padding related content

Reply:
For padding its fine, but for width its preferable to use something generic as rem, spacing from theme may differ from project to project, this will help in getting more collaborators and frontend developers onboard by making the styles easy interpret.

Reply:
@ishangupta-ds  ok thanks!